user/models.py

```diff
@@ -1,3 +1,7 @@
 from django.db import models
 
-# Create your models here.
+class User(models.Model):
+    id = models.CharField(max_length=255, primary_key=True)
+    username = models.TextField()
+    password = models.TextField()
+    email = models.TextField()
```

Comment:
Does this site require having username distinct from email?

If you derive from `AbstractBaseUser` and declare `email` to be the `USERNAME_FIELD` then `UserCreationForm` will already include it.

Reply:
Yeah, i think ONLY usernameis fine. we dont need both anymore